crates/holochain_zome_types/src/element.rs

```diff
@@ -251,49 +295,45 @@ impl std::hash::Hash for SignedHeaderHashed {
     }
 }
 
-impl PartialEq for SignedHeaderHashed {
-    fn eq(&self, other: &Self) -> bool {
-        self.signature.eq(&other.signature) && self.as_hash() == other.as_hash()
-    }
-}
-
-#[allow(missing_docs)]
 impl SignedHeaderHashed {
```

Comment:
I think these can all be generic

```suggestion
impl<T> SignedHashed<T> {
```